bookkeeper-server/src/test/java/org/apache/bookkeeper/server/http/TestHttpService.java

```diff
@@ -769,10 +769,11 @@ public void testTriggerGCService() throws Exception {
         HttpEndpointService triggerGCService = bkHttpServiceProvider
             .provideHttpEndpointService(HttpServer.ApiType.GC);
 
-        //1,  GET, should return NOT_FOUND
+        //1,  GET, should return OK
         HttpServiceRequest request1 = new HttpServiceRequest(null, HttpServer.Method.GET, null);
         HttpServiceResponse response1 = triggerGCService.handle(request1);
-        assertEquals(HttpServer.StatusCode.NOT_FOUND.getValue(), response1.getStatusCode());
+        assertEquals(HttpServer.StatusCode.OK.getValue(), response1.getStatusCode());
+        assertTrue(response1.getBody().contains("false"));
```

Comment:
shall we use "equals" instead of "contains" to make the assertion more explicit?

Reply:
Thanks. the response body contains other fixed words, the only change-able word are false/true.